breathecode/events/tasks.py

```diff
@@ -199,3 +202,74 @@ def fix_live_class_dates(timeslot_id: int):
 
         starting_at += delta
         ending_at += delta
+
+
+@task(priority=TaskPriority.ACADEMY.value)
+def generate_event_recap(event_id: int, **kwargs):
+    """
+    Generate a recap of the event using rigobot AI.
+    This task will be triggered when an event changes to FINISHED status.
+    """
+    import re
+
+    event = Event.objects.filter(id=event_id).first()
+    if not event:
+        logger.error(f"Event {event_id} not found")
```

Comment:
`AbortTask` and the another Task Manager's exceptions print this error https://github.com/breatheco-de/celery-task-manager-django-plugin/blob/3bafe5b2dfa2a4b55b708f89e0c95c5216737c3a/src/task_manager/core/decorators.py#L276